solution/backend/regulations/templates/regulations/statute.html

```diff
@@ -0,0 +1 @@
+<p>hello world</p>
```

Comment:
We shouldn't see the "no newline at end of file" anymore

Reply:
updated